File: Tests/CSharp/Underscores/Underscores.cs
// **********************************************************************
//
// Copyright (c) 2009-2015 ZeroC, Inc. All rights reserved.
//
// **********************************************************************

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Test
{
    public class Underscores
    {
        public static bool underscores()
        {
            Person p = new Person();
            p.p_name = "Foo";
            p.p_email = "foo@bar.org";
            return true;
        }
    }
}

File: Tests/Cpp/Underscores/Underscores.cpp
// **********************************************************************
//
// Copyright (c) 2009-2016 ZeroC, Inc. All rights reserved.
//
// **********************************************************************

#include <Ice\Ice.h>
#include <Test.h>

#include "Underscores.h"

bool Test::underscores()
{
	Test::PersonPtr p = new Test::Person();
	p->p_name = "Foo";
	p->p_email = "foo@bar.org";
	return true;
}